keps/sig-storage/20190408-volume-scheduling-limits.md

```diff
@@ -17,7 +17,7 @@ approvers:
 editor: TBD
 creation-date: 2019-04-08
 last-updated: 2019-04-08
-status: implementable
+status: implemented
```

Comment:
Can you update the "Implementation History" section too?

Reply:
Updated